src/Randomizer.App/Windows/GenerateRomWindow.xaml

```diff
@@ -7,20 +7,20 @@
         xmlns:local="clr-namespace:Randomizer.App"
         xmlns:controls="clr-namespace:Randomizer.App.Controls"
         xmlns:vm="clr-namespace:Randomizer.App.ViewModels"
-        xmlns:smz3="clr-namespace:Randomizer.SMZ3;assembly=Randomizer.SMZ3"
+        xmlns:data="clr-namespace:Randomizer.Data;assembly=Randomizer.Data"
         xmlns:shared="clr-namespace:Randomizer.Shared;assembly=Randomizer.Shared"
-        d:DataContext="{d:DesignInstance vm:RandomizerOptions, IsDesignTimeCreatable=True}"
+        xmlns:options="clr-namespace:Randomizer.Data.Options;assembly=Randomizer.Data"
         mc:Ignorable="d"
         Closing="Window_Closing"
-        Title="SMZ3 Cas' Randomizer"
+        Title="Data Cas' Randomizer"
```

Comment:
I think you might've Found and Replaced a little bit too much

Reply:
Good catch. I think this was probably a single file find & replace as a global one like that would have been real dumb plus I think we would have seen a lot more. However, just to be safe I'll review over the changes myself again later when I have time to make sure nothing else like this happened though as a bit of due diligence.

Reply:
Yeah, I don't think Data Cas’ Randomizer is the new name we wanted to go for, haha